### PR TITLE
MAT-5253: Add supplemental data and risk adjustment

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE PR Template

Jira Ticket: [MAT-5253](https://jira.cms.gov/browse/MAT-5253)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by mcmcphillips — DISMISSED** [review]

File: src/main/java/gov/cms/madie/models/measure/Measure.java [hunk 1]
```diff
@@ -106,6 +106,8 @@ public class Measure extends ResourceAcl {
   private String lastModifiedBy;
   private Date measurementPeriodStart;
   private Date measurementPeriodEnd;
+  private List<SupplementalData> supplementalData;
+  private List<RiskAdjustment> riskAdjustments;
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/Measure.java:110]: for some reason, I remember @sb-cecilialiu also working on a similar thing.  
Also, I don't think we need two different models for this. they are both supplemental data elements with small distinction. I was thinking if we could do this

List<SepplimentalDataElement> sepplimentalDataElements;
where SepplimentalDataElement is,
SepplimentalDataElement {
id,
cqlDefinition: string
description: string
type: Enum(supplemental-data | risk-adjustment-factor)
}  

what do you think? cc:@jkotanchik-SB , @RohitKandimalla

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/models/measure/Measure.java:110]: I believe it will be the same for QDM as well, so I don't have anything against having a single structure. LGTM

Comment (jkotanchik-SB) [reply src/main/java/gov/cms/madie/models/measure/Measure.java:110]: I'm fond of readability when all other factors being equal. Unless we're gaining a decent benefit from combining the two, I'd prefer they stay separate.

Based on what @adongare said about their relationship, I'm thinking RAV could be nearly empty subclass of SDE to capture that relationship.

Comment (adongare) [reply src/main/java/gov/cms/madie/models/measure/Measure.java:110]: Okay, separate models then.

**Review by adongare — DISMISSED** [review]

**Review by jkotanchik-SB — APPROVED** [review]

**Review by jkotanchik-SB — APPROVED** [review]
